src/_components/ui/ParticlesAnimation.tsx

```diff
@@ -0,0 +1,33 @@
+import { hooks, renderToString } from "lume/deps/preact.ts";
+import { loadFull } from "npm:tsparticles";
+import options from "./particles.options.json" assert { type: "json" };
+import Particles from "npm:preact-particles";
+import type { Container, Engine } from "npm:tsparticles-engine";
+
+export default () => {
+  const particlesInit = hooks.useCallback(async (engine: Engine) => {
+    console.log(engine);
+    // you can initiate the tsParticles instance (engine) here, adding custom shapes or presets
+    // this loads the tsparticles package bundle, it's the easiest method for getting everything ready
+    // starting from v2 you can add only the features you need reducing the bundle size
+    await loadFull(engine);
+  }, []);
+
+  const particlesLoaded = hooks.useCallback(async (container: Container) => {
+    await console.log(container);
+  }, []);
+
+  const fragment = (
+    <>
+      <Particles
```

Comment:
Particles is not recognized : 
![tsparticles_not_jsx_element](https://github.com/severindupouy/severin-dupouy.deno.dev/assets/32488190/a682a7fd-b432-48e7-b25c-ce866f13bc77)
And when I run the app :
![tsparticles_failing_to_serve](https://github.com/severindupouy/severin-dupouy.deno.dev/assets/32488190/e2444384-af50-4421-b06d-d93274df01b6)